External/FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -1182,8 +1182,10 @@ void *JITCore::CompileCode([[maybe_unused]] FEXCore::IR::IRListView<true> const
         auto Op = IROp->C<IR::IROp_Ashr>();
         if (OpSize == 8)
           asrv(GetDst<RA_64>(Node), GetSrc<RA_64>(Op->Header.Args[0].ID()), GetSrc<RA_64>(Op->Header.Args[1].ID()));
-        else
-          asrv(GetDst<RA_32>(Node), GetSrc<RA_32>(Op->Header.Args[0].ID()), GetSrc<RA_32>(Op->Header.Args[1].ID()));
+        else {
```

Comment:
How about in the future we do `else if (size == XX || YY || ZZ) { } else { assert(bad stuff); }` so the range of supported values is visible?